drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -133,6 +133,36 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
     joins(:bed_nights).where(s_t[:date_provided].eq(date))
   end
 
+  # @param range [DateRange]
+  # enrollments that conflict with an entry/exit date
+  # * entry date on exit date is allowed
+  # * multiple entry dates on same day are not allowed
```

Comment:
This is only absolutely true for the same project, or maybe for the same residential project type.  There's no restriction that would prevent you from having an overlapping project in ES and Services only or CE for example.  Also, you can totally have two Service Only enrollments open at the same time.

Reply:
@eanders , you commented [here](https://www.pivotaltracker.com/story/show/185728844/comments/237822714) that its explicitly forbidden by HUD for a client to be enrolled multiple times at the same project. Are you saying that's only for certain project types? I can't find any guidance on this, but I don't see why someone would need to be doubly enrolled at a project.

Reply:
Nope, that's correct.  It is forbidden to be enrolled at the same PROJECT, but you CAN be enrolled in the same project type, as long as it's not a residential project (ES, TH, SH, PH).

Reply:
Oh, we don't have any restrictions around that yet. I wasn't aware of that. This PR is only checking for being enrolled at the same project twice.